.github/workflows/ci.yml

```diff
@@ -165,3 +166,9 @@ jobs:
 
       - name: Lint Toml
         run: pnpm lint-toml
+
+      - name: update generated code
+        run: |
+          cargo run --bin generator
```

Comment:
Would it be better if this process it's part of `cargo build` or `pnpm build`? Is it quite slow?

Reply:
It is not slow, but since it needs to compile first, and for now the generated code should not be changed frequently, So I don't want to slow down the normal workflow, Putting it here could ensure the generated code is always up to date, also don't bother developer's normal workflow?

Reply:
We could modify it in the future if we add some generators that change frequently, perhaps like napi binding code getter/setter used for syncing data between Rust and JavaScript.